.bashrc

```diff
@@ -58,16 +58,14 @@ export HISTFILE=${HOME}/.bash_eternal_history
 #     fi
 # fi
 
+export PROMPT_COMMAND='export last_exit="$?"; history -a; history -c; history -r; __prompt_command $last_exit'
 
-function __exit_stat() {
-  stat=$?;
-  echo $stat
+function parse_git_branch() {
+         git branch 2> /dev/null | sed -e '/^[^*]/d' -e 's/* \(.*\)/ (\1)/'
 }
 
-export PROMPT_COMMAND="__prompt_command $(__exit_stat)"# Func to gen PS1 after CMDs
-
 function __prompt_command() {
-    local Exit="$1"
+    local Exit=$1
```

Comment:
Quotes meant it got eval'd during script load. instead of when function ran. of freaking course..... HOURS LOST DEBUGGING THIS SHITE